src/table/column-filter.scss

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-column-filter {
+  @include iui-dropdown;
+  padding: $iui-baseline $iui-sm;
+  width: fit-content;
+
+  .iui-overlay {
+    position: relative;
+
+    > .iui-input {
+      padding-right: $iui-l * 2;
+    }
+
+    > .iui-borderless {
+      position: absolute;
+      right: 0;
+    }
+  }
+
+  .iui-right-align {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+
+    > .iui-date-time {
+      display: flex;
+
+      > .iui-input-container > .iui-input {
+        width: $iui-3xl * 2;
+      }
+
+      > .iui-input-container + .iui-input-container {
+        margin-left: $iui-s;
```

Comment:
Maybe margin-right with `:not(:last-of-type)` for clarity?

Reply:
Might not work actually 🤔

Reply:
It does work for the margin but I'd have to add an extra class to achieve the width applied to the time text field.

Reply:
Yeah I guess don't worry about it